eng/Versions.props

```diff
@@ -139,7 +139,7 @@
     <LinuxCoreFxOptimizationDataPackage>optimization.linux-x64.ibc.corefx</LinuxCoreFxOptimizationDataPackage>
     <MicrosoftPrivateIntellisensePackage>microsoft.private.intellisense</MicrosoftPrivateIntellisensePackage>
     <MicrosoftDotNetXUnitConsoleRunnerPackage>Microsoft.DotNet.XUnitConsoleRunner</MicrosoftDotNetXUnitConsoleRunnerPackage>
-    <MicrosoftNETCoreAppPackage>Microsoft.NETCore.App</MicrosoftNETCoreAppPackage>
+    <MicrosoftNETCoreAppPackage>$(SharedFrameworkName)</MicrosoftNETCoreAppPackage>
```

Comment:
These props are truly for the package names, so I don't think this is necessary. Maestro++ might even revert this automatically, but I don't remember for sure.

(This corresponds to the package ID for `<MicrosoftNETCoreAppVersion>5.0.0-preview.4.20202.18</MicrosoftNETCoreAppVersion>`.)

Reply:
Ah ok, reverted this change